libs/community/langchain_community/chains/pebblo_retrieval/utilities.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import os
+import platform
+from typing import Tuple
+
+from langchain_core.env import get_runtime_environment
+
+from langchain_community.chains.pebblo_retrieval.models import Framework, Runtime
+
+logger = logging.getLogger(__name__)
+
+PLUGIN_VERSION = "0.1.1"
+
+CLASSIFIER_URL = os.getenv("PEBBLO_CLASSIFIER_URL", "http://localhost:8000")
+PEBBLO_CLOUD_URL = os.getenv("PEBBLO_CLOUD_URL", "https://api.daxa.ai")
+
+PROMPT_URL = "/v1/prompt"
+APP_DISCOVER_URL = "/v1/app/discover"
+
+
+def get_runtime() -> Tuple[Framework, Runtime]:
+    """Fetch the current Framework and Runtime details.
+
+    Returns:
+        Tuple[Framework, Runtime]: Framework and Runtime for the current app instance.
+    """
+    runtime_env = get_runtime_environment()
+    framework = Framework(
+        name="langchain", version=runtime_env.get("library_version", None)
```

Comment:
Why do we need `None` here? 
Also, since `version` is defined as a `str` in the Framework model, `None` is not going to work with it.